cookbook/examples/apps/gemini-edu/requirements.txt

```diff
@@ -0,0 +1,15 @@
+agno>=0.1.0
```

Comment:
Check how the other apps work with `requirements.in` and generate_requirements.sh, please add these rather.